resources/lang/en/Widget.properties

```
@@ -465,3 +465,5 @@ orderHistoryAllowPaymentProviderChangeLabel = "Allow customer to change the paym
 orderHistoryAllowPaymentProviderChangeTooltip = "Allow customer to change the payment method."
 orderHistoryAllowReturnLabel = "Allow returns"
 orderHistoryAllowReturnTooltip = "Allow returns."
+; Tab Widget
+tabLabel = "Tab Widget"
```

```suggestion
tabLabel = "Tabs"
```

Wie oben von Felix angemerkt